include/triton/Conversion/TritonToTritonGPU/Passes.td

```diff
@@ -29,8 +29,8 @@ def ConvertTritonToTritonGPU: Pass<"convert-triton-to-tritongpu", "mlir::ModuleO
               "int32_t", /*default*/"1",
               "number of ctas in a cga">,
         Option<"computeCapability", "compute-capability",
-              "int32_t", /*default*/"80",
-              "compute capability">
+              "std::optional<int32_t>", /*default*/"std::nullopt",
+              "optional compute capability for cuda">
```

Comment:
that seems a bit odd, we could either remove this from the metadata or make it a generic triple? What do you think?

Reply:
yup that makes sense. planning to do that as the next step. for this pull request just focusing on getting the amd backend target story in place.